uae4all_gp2x_0.7.2a/src/linetoscr.h

```diff
@@ -8,31 +8,55 @@ static __inline__ int LNAME (int spix, int dpix, int stoppos)
     if (bpldualpf) {
 	    // OCS/ECS Dual playfield 
 	    int *lookup = bpldualpfpri ? dblpf_ind2 : dblpf_ind1;
-		
-	    while (dpix < stoppos) {
+		int n = (stoppos-dpix);
+    
+        //center fix mithrendal
+        dpix = (dpix-VISIBLE_LEFT_BORDER)*2 + VISIBLE_LEFT_BORDER;
+	    while (n--) {
 			register unsigned short d = colors_for_drawing.acolors[lookup[pixdata.apixels[spix]]];
 			buf[dpix++] = d;
-			spix += SRC_INC;
+            buf[dpix++] = d;
+            spix ++;
 	    }
 		
```

Comment:
found in xcode prefences the identation settings. Before mine was "prefer indents using spaces".. I switched it now to "prefer indents using tabs" and commited the file again.  ;-)